### ISSUE TITLE
PHP Issue when running initial docker commang

### ISSUE THREAD

**Issue by bnornholm** [shlinkio/shlink#561]
When running the initial docker run command, I get an error of:

`In ProcessHelper.php line 106:
The command /usr/local/bin/php vendor/doctrine/migrations/bin/doctrine-migrations.php migrations:migrate failed.`

`Error Output:
In YamlNotAvailable.php line 13:
Unable to load yaml configuration files, please run composer require symfony/yaml to load yaml configuration files.`

I ran the composer command but the issue still persists. I have php 7.3 installed the the necessary extensions.  Any ideas?

**Comment by atx-coder** [comment]
Same:


```
[2019-12-05T19:54:14.079752+00:00] Shlink.INFO - Successfully acquired the "db:create" lock.
[2019-12-05T19:54:14.079855+00:00] Shlink.INFO - Expiration defined for "db:create" lock for "90" seconds.
Updating database...
[2019-12-05T19:54:14.482261+00:00] Shlink.INFO - Successfully acquired the "db:migrate" lock.
[2019-12-05T19:54:14.482408+00:00] Shlink.INFO - Expiration defined for "db:migrate" lock for "90" seconds.

In ProcessHelper.php line 106:

  The command "'/usr/local/bin/php' 'vendor/doctrine/migrations/bin/doctrine-
  migrations.php' 'migrations:migrate'" failed.

  Exit Code: 1(General error)

  Working directory: /etc/shlink

  Output:
  ================


  Error Output:
  ================

  In YamlNotAvailable.php line 13:


    Unable to load yaml configuration files, please run `composer require sym
  fo
    ny/yaml` to load yaml configuration files.




  migrations:migrate [--write-sql [WRITE-SQL]] [--dry-run] [--query-time] [--
  allow-no-migration] [--all-or-nothing [ALL-OR-NOTHING]] [--configuration [C
  ONFIGURATION]] [--db-configuration [DB-CONFIGURATION]] [--] [<version>]


db:migrate

```

**Comment by acelaya** [comment]
Ugh. This is annoying.

What shlink version are you guys running?

It's probably easy to fix, but should not be happening. I will investigate and provide a fix with next release.

**Comment by bnornholm** [comment]
Technically not running any version currently, but using the most current version.  

When that error comes up during `docker run --name shlink -p 8080:8080 -e SHORT_DOMAIN_HOST=doma.in -e SHORT_DOMAIN_SCHEMA=https shlinkio/shlink`, it exits and the container enters and exited state and wont let it get past it.

**Comment by acelaya** [comment]
Ok. If it's not part from a stable release, then it's not that big a deal.

Can you try `docker run --name shlink -p 8080:8080 -e SHORT_DOMAIN_HOST=doma.in -e SHORT_DOMAIN_SCHEMA=https shlinkio/shlink:stable`?

That one will get you the latest stable version.

**Comment by acelaya** [comment]
I haven't verified affected versions, but the issue is already fixed and will be part of the upcoming v1.20.2

**Comment by bnornholm** [comment]
Fixed by using the `:stable` tag

**Comment by acelaya** [comment]
Good yo know that stable versions are not affected.

Thanks for the info.

**Comment by acelaya** [comment]
I have just released v1.20.2

Fixes have also been added to `latest`, which is being built right now.
